### PR TITLE
Renamed domainName to domains

### PR DESCRIPTION
**Pull request by RockMother**
Renamed domainName to domains

### CODE REVIEW

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -6140,8 +6140,14 @@ components:
           type: string
           description: The company name.
         domainName:
+          deprecated: true
```
Comment (Zemidar) [inline sell/sell-api.yaml:6143]: What about add info about deprecation and new implementation to description?

**Review by Zemidar — APPROVED** [review]

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -6140,8 +6140,14 @@ components:
...
           type: string
           description: The website.
+        domains:
+          type: array
+          items:
+            type: string
+          description: The website separated by comma.
```
Comment (olyavertwist) [inline sell/sell-api.yaml:6150]: description: The list of website URLs separated by commas.

**Review by olyavertwist — CHANGES_REQUESTED** [review]

File: sell/sell-api.yaml (hunk 2 above)
Comment (RockMother) [reply sell/sell-api.yaml]: Done

File: sell/sell-api.yaml (hunk 1 above)
Comment (RockMother) [reply sell/sell-api.yaml:6143]: Nice idea!

**Review by olyavertwist — APPROVED** [review]

File: sell/sell-api.yaml (hunk 1 above)
Comment (eprudnikov) [reply sell/sell-api.yaml:6143]: @RockMother , thanks a lot for keeping it compatible!

**Pull request merged**